docs/user-guide.md

```diff
@@ -574,3 +633,11 @@ After reviewing the downloaded report and verifying its content, the admin can c
 * After selecting the checkbox, the admin will be presented with a comment box to add mandatory comments, specifying the reason for rejection. After providing the required comments, the admin can proceed by clicking the `Reject` button to officially reject the report.
 
 ![CTK Partner Report Dashboard](\_images/ctk-partnerReports-reject.png)
+
+### Test Runs Archival Batch Job
+
+Compliance Toolkit has a batch job process that archives X number of test runs for each collection.
+
+For example, if there are 15 test runs with an offset of 10, the most recent 10 will be retained, and the remaining 5 will be moved to archival table. In `test-run-history`, only the last 10 test run records are visible to partners.
```

Comment:
Consider this scenario: If there are 15 test runs, with an offset of 10, the most recent 10 will be retained, while the remaining 5 will be moved to an archival table. In the test-run-history, only the last 10 test run records are visible to partners.